src/elements/content-sidebar/activity-feed/task-new/__tests__/Task-test.js

```diff
@@ -107,23 +107,23 @@ describe('elements/content-sidebar/ActivityFeed/task-new/Task', () => {
                 currentUser={currentUser}
                 onEdit={jest.fn()}
                 onDelete={jest.fn()}
-                due_at={new Date() - 1000}
+                due_at={775090439000}
                 status="NOT_STARTED"
             />,
         );
-        expect(incompleteWrapper.find('.bcs-task-overdue')).toHaveLength(1);
+        expect(incompleteWrapper).toMatchSnapshot();
```

Comment:
the snapshot is pretty broad for what is asserted in this test - maybe verify the mock comment  is passed the formatted timestamp or snapshot the specific prop